web/docs/Frontend Starters/Gatsby/Gatsby + WordPress/setting-environment-variables.md

```diff
@@ -45,20 +43,16 @@ let url = process.env.WPGRAPHQL_URL || process.env.PANTHEON_CMS_ENDPOINT;
 ```
 
 In order to connect to a Multidev backend, this url will need to be updated.
-`PANTHEON_ENVIRONMENT_URL` includes the PR number or integration branch name and
-you can parse it to your needs.
+`PANTHEON_ENVIRONMENT` includes a PR number or integration branch name.
 
 This code could be added under the above logic to connect to a Multidev that is
 prefixed with the branch name of my site.
 
 ```js
 /**
-* My branch is named `multi-demo`. I will parse the environment url for that substring
-* and use that, along with the `url`, to create a variable which points
-to my Multidev backend.
-**/
-const PREFIX = process.env.PANTHEON_ENVIRONMENT_URL.match(/^([^-]*-)[^-]*/)[0];
-if (!process.env.IS_LIVE_ENVIRONMENT) {
-	url = `https://${PREFIX}-${url.replace(/^https?:\/\/[^-]*-/, '')}`;
+ * PANTHEON_ENVIRONMENT is equal to `multi-demo` since that is the name of my branch.
+ **/
+if (PANTHEON_ENVIRONMENT !== 'live') {
+	url = `https://${PANTHEON_ENVIRONMENT}-${url.replace(/^https?:\/\//, '')}`;
```

Comment:
This assumes the `url` doesn't already have a multidev prefix like `dev-` or `live-`, correct? Wouldn't we still need to remove `dev-` or whatever from the `url`?

Reply:
Ah yes, you are correct. The WPGRAPHQL_URL I was using on my old site was uploaded without the prefix so that's why this was passing in my testing. Updating now 👍